<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>411</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Habits</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/normalWindowsUser/Desktop/Klotman.png</normaloff>C:/Users/normalWindowsUser/Desktop/Klotman.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>90</y>
      <width>577</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your habit list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(159, 159, 159);</string>
    </property>
   </widget>
   <widget class="QLabel" name="menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/mw/res/mw/MainWindow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(240, 240, 240);
font: 75 11pt &quot;Arial&quot;;
font: 87 8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/add.png</normaloff>:/mw/res/mw/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="viewButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(240, 240, 240);
font: 75 11pt &quot;Arial&quot;;
font: 87 8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/today.png</normaloff>:/mw/res/mw/today.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>151</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(240, 240, 240);
font: 75 11pt &quot;Arial&quot;;
font: 87 8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/edit.png</normaloff>:/mw/res/mw/edit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>202</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(240, 240, 240);
font: 75 11pt &quot;Arial&quot;;
font: 87 8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/remove.png</normaloff>:/mw/res/mw/remove.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="habitListLbl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/habitlist.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="titleLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/mw/res/mw/title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="optionsLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/mw/res/mw/options.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otherLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/mw/res/mw/other.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/about.png</normaloff>:/mw/res/mw/about.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>350</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="statisticsB">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>360</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(240, 240, 240);
font: 75 11pt &quot;Arial&quot;;
font: 87 8pt &quot;Arial Black&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/Stat.png</normaloff>:/mw/res/mw/Stat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="closeB">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/close.png</normaloff>:/mw/res/mw/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>190</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeB">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/mw/res/mw/minimize.png</normaloff>:/mw/res/mw/minimize.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>190</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>menu</zorder>
   <zorder>minimizeB</zorder>
   <zorder>list</zorder>
   <zorder>addButton</zorder>
   <zorder>viewButton</zorder>
   <zorder>modifyButton</zorder>
   <zorder>deleteButton</zorder>
   <zorder>habitListLbl</zorder>
   <zorder>titleLbl</zorder>
   <zorder>optionsLbl</zorder>
   <zorder>otherLbl</zorder>
   <zorder>aboutB</zorder>
   <zorder>statisticsB</zorder>
   <zorder>closeB</zorder>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
